### PR TITLE
Added single API method of "editBug"

### PR DESCRIPTION
**Pull request by jeffreypriebe**
1. Added editBug from: https://github.com/wangyangjun/node-fogbugz/commit/3797cb7993c81825d166517b1766b56dc641ed33
    * Including tests
2. Fixed a small bug in tests

### CODE REVIEW

**Comment by boneskull** [comment]
Ty for the PR. I'm traveling but will check on this over the weekend (or earlier). Remind me if I forget!

**Comment by jeffreypriebe** [comment]
You're welcome. Thanks for starting this project - I got an internal sync tool off the ground more quickly (and didn't have to look at any XML from the API).

**Comment by jeffreypriebe** [comment]
Reminder re-ping.

**Comment by boneskull** [comment]
@jeffreypriebe Yeah, who doesn't love parsing XML?  :P  These old APIs just cry out for an abstraction layer

File: spec/fogbugz.spec.js [hunk 1]
```diff
@@ -60,8 +105,8 @@ describe('fogbugz module', function () {
     });
   });
 
-  describe('logoff method', function (done) {
-    it('should fail w/o presence of token', function () {
+  describe('logoff method', function () {
+    it('should fail w/o presence of token', function (done) {
```
Comment (boneskull) [inline spec/fogbugz.spec.js]: thanks for this.  pretty old module.  if I had my druthers I'd rewrite it in mocha, but whatever

**Comment by boneskull** [comment]
OK, I've taken a look at this and I have a couple notes:

1.  Please remove anything to do with TypeScript
2.  Rebase + squash on top of `master`, then `npm install` and re-run `npm test`

It appears I didn't have any guidelines here around code style, so I've added them to ensure consistency.  I was seeing inconsistent indents in the PR, so I figured I should do something about it.

**Comment by boneskull** [comment]
Thanks again

**Comment by jeffreypriebe** [comment]
New PR: https://github.com/boneskull/node-fogbugz/pull/13
(This got closed in my noob branching/rebasing)

**Pull request closed without merging**